### PR TITLE
Fix pathfinding

### PR DESCRIPTION
**Pull request by Riddarvid**

### CODE REVIEW

File: game/core/src/main/java/com/gg/game/services/pathFinding/PointGraph.java [hunk 1]
```diff
@@ -84,34 +81,33 @@ private boolean hasLineOfSight(PfaPoint p1, PfaPoint p2, List<Collidable> collid
...
-        Point p1 = collidable.getPosition();
-        Point p2 = p1.moveBy(widthVector);
-        Point p3 = p1.moveBy(heightVector);
-        Point p4 = p1.moveBy(widthVector).moveBy(heightVector);
-        p1 = p1.moveBy(new Vector(-margin, -margin));
-        p2 = p2.moveBy(new Vector(margin, -margin));
-        p3 = p3.moveBy(new Vector(-margin, margin));
-        p4 = p4.moveBy(new Vector(margin, margin));
+        Point p1 = center.moveBy(widthVector).moveBy(heightVector);
+        Point p2 = center.moveBy(widthVector).moveBy(heightVector.scale(-1f));
```
Comment (heinlson) [inline game/core/src/main/java/com/gg/game/services/pathFinding/PointGraph.java]: -1f? Is it supposed to be -margin?

File: game/core/src/main/java/com/gg/game/model/Character.java [hunk 2]
```diff
@@ -89,6 +89,11 @@ public float getRotation() {
         return 0;
     }
 
+    @Override
+    public Point getCenter() {
```
Comment (fourtooths) [inline game/core/src/main/java/com/gg/game/model/Character.java]: Kodduplicering?

File: game/core/src/main/java/com/gg/game/services/pathFinding/PointGraph.java (hunk 1 above)
Comment (Riddarvid) [reply game/core/src/main/java/com/gg/game/services/pathFinding/PointGraph.java]: No, -1f is to flip the vector

File: game/core/src/main/java/com/gg/game/model/Character.java (hunk 2 above)
Comment (heinlson) [reply game/core/src/main/java/com/gg/game/model/Character.java]: where is the other duplicate?

Comment (Riddarvid) [reply game/core/src/main/java/com/gg/game/model/Character.java]: This has been fixed now

**Pull request merged**